src/legacy/server/kbn_server.d.ts

```diff
@@ -101,38 +105,63 @@ export interface PluginsSetup {
   [key: string]: object;
 }
 
+export interface Platform {
+  env: {
+    mode: Readonly<EnvironmentMode>;
+    packageInfo: Readonly<PackageInfo>;
+  };
+  handledConfigPaths: UnwrapPromise<ReturnType<ConfigService['getUsedPaths']>>;
```

Comment:
As the type was properly extracted, should we move it to `core/server/legacy/somewhere` ?

Reply:
Could you elaborate on what you mean by this?

Reply:
As this interface is related to NP, I was just asking if we should move it somewhere in core and import it in `kbn_server.d.ts` instead. Could then be also used to type which is currently `any` https://github.com/elastic/kibana/blob/ba9a6c91955dddb2d5d49546a9250b6dd5ab0210/src/core/server/legacy/legacy_service.ts#L286-L312

Reply:
Looks like this was cleaned up elsewhere, so resolving. :+1: